### ISSUE TITLE
Create backend structure

### ISSUE THREAD

**Issue by Malaguth** [Malaguth/skill-tracking#37]
# User Story

As a backend developer, I want to set up the project structure for a Node.js application using TypeScript and MVC architecture so that I can start building the application with a well-organized, scalable, and maintainable codebase.

## Description

This user story focuses on the initial setup of the project structure for a Node.js application using TypeScript and MVC (Model-View-Controller) architecture. The goal is to create a robust and efficient development environment that encourages separation of concerns and clean code practices. This includes setting up the basic directory structure following MVC principles, installing necessary dependencies, and configuring TypeScript for the project.

## Acceptance Criteria

1. A new repository should be created for the project.
2. The repository should be initialized with a package.json file by running `npm init`.
3. Node.js and Express should be installed in the project as dependencies.
4. TypeScript should be installed as a devDependency.
5. A tsconfig.json file should be created and configured according to the project's needs.
6. An MVC directory structure should be set up which includes at least:
   - models folder: inside src, for all data-related logic
   - views folder: inside src, for the user interface (not typically used in API-only applications but can be included for server-side rendered views)
   - controllers folder: inside src, for handling user requests and returning responses
7. Example files for each part of MVC (model, view, controller) should be created using TypeScript.
8. The application should run without errors when the command `npm start` is executed.
9. The project structure and setup, including the use of MVC architecture, should be documented in the README file.

**Comment by Malaguth** [comment]
@jpbahiaz, could you help me define if using npm or another manager as yarn is better? Also, I think the current LTS version is the best option. Do you agree?

**Comment by jpbahiaz** [comment]
I agree. I'd suggest using pnpm as a package manager but I think we should stick with npm for simplicity.
